IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -198,11 +187,7 @@ - (BOOL)obtainParentController
     
     if (parentController) return YES;
     
-    if (@available(iOS 13.0, *)) return NO;
-    
-    parentController = [UIApplication msidCurrentViewController:parentController];
```

Comment:
I'm curious what was the previous code doing?

Reply:
Previous code on below ios13 was trying to get rootWebviewWithParentController  (r(if parentController is not nil, we return YES before hitting this call) and if no parentController get keywindow. And if ios13, prev code was returning NO